tools/cli/internal/apiversion/version.go

```diff
@@ -247,3 +291,80 @@ func Sort(versions []*APIVersion) {
 		}
 	}
 }
+
+// GetPreviewVersionName returns the preview version name.
+func GetPreviewVersionName(contentTypeValue *openapi3.MediaType) (name string, err error) {
```

Comment:
Does this need to be public? 👀 
```suggestion
func getPreviewVersionName(contentTypeValue *openapi3.MediaType) (name string, err error) {
```

Reply:
I think yes cause we use it in the versions package